lib/internal/errors.js

```diff
@@ -22,6 +23,7 @@ const {
   SymbolFor,
   WeakMap,
 } = primordials;
+const sep = process.platform === 'win32' ? '\\' : '/';
```

Comment:
isn't this already `path.sep`? it'd be nice to reuse that.

Reply:
Sure, I've referenced path for this.

Reply:
This can't actually be referenced due to a circular definition ordering issue between the files. Error codes must be defined before path. So we end up having to move the require to the end of the file, and not destructuring sep, or just inlining the definition of sep. I've gone back to inlining, but if you feel strongly about this we can do the former too.

Reply:
oof, ok - maybe there's some larger refactoring needed to avoid that circular dep while maintaining a single source of truth, but that seems like a task for a future PR; inlining should be fine for now.